reviews/sandbox/2022/2022-Confidential-Containers-annual.md

```diff
@@ -0,0 +1,155 @@
+
+The project hourly activity for the same period is shown in the below chart:
+* [Confidential Containers GitHub Hourly Activity (2022-03-08 to Now)](https://confidentialcontainers.devstats.cncf.io/d/8/dashboards?orgId=1&from=1646694000000&to=now-1h&viewPanel=2&refresh=15m)
+
+Confidential Containers new Contributors:
+* [Confidential Containers New Contributors (2022-03-08 to Now)](https://confidentialcontainers.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=1646694000000&to=now-1h)
+
+These charts and statistics show a few interesting points:
+
+* Since its acceptance as a CNCF project, the project has seen a steady stream
+  of contributions.
+* A very important metric for Confidential Containers: More than 40 new
+  contributors have joined the project over the past year.
+* Contributions are spread over different repositories, showing the width of the
+  project:
+  * On the client side, the `enclave-cc`, `attestation-agent`, `td-shim` and
+    `image-rs` projects take the bulk of the contributions.
+  * On the attestation part of the project, most of the contributions went to
+    the `kbs` and `attestation-service` repositories.
+  * The `cloud-api-adaptor` and `operator` sub-projects merged most of the
+    infrastructure related contributions to Confidential Containers.
+
+## Maintainers
+
+The Confidential Containers project maintainership structure is defined as
+follows:
+
+* Each sub-project has its own list of maintainers, tracked through both a
+  per-repo MAINTAINERS file and a corresponding GitHub team. We have multiple
+  instances of individuals maintaining several sub-projects.
+* A Technical Steering Committee (TSC) provides decision-making and strategic
+  oversight for the project.
+
+The [TSC](https://github.com/confidential-containers/community/blob/main/MAINTAINERS)
+is composed of individuals from multiple companies (Alibaba, AMD, IBM, Intel,
+Red Hat and Rivos):
+
+- [@ariel-adam](https://github.com/ariel-adam), Ariel Adam, RedHat
+- [@bpradipt](https://github.com/bpradipt), Pradipta Banerjee, IBM
+- [@dcmiddle](https://github.com/dcmiddle), Dan Middleton, Intel
+- [@fitzthum](https://github.com/fitzthum), Tobin Feldman-Fitzthum, IBM
+- [@jiazhang0](https://github.com/jiazhang0), Zhang Jia, Alibaba
+- [@jiangliu](https://github.com/jiangliu), Jiang Liu, Alibaba
+- [@larrydewey](https://github.com/larrydewey), Larry Dewey, AMD
+- [@magowan](https://github.com/magowan), James Magowan, IBM
+- [@peterzcst](https://github.com/peterzcst), Peter Zhu, Intel
+- [@sameo](https://github.com/sameo), Samuel Ortiz, Rivos
+
+## Adoption
+
+> **Note**
+> From CNCF: What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are adopters of your project, please do so._
+
+Cloud Providers (Microsoft, Alibaba, IBM) are pusuring offerings which will
+include Confidential Containers. They are actively engaged in the community.
+
```

Comment:
@ariel-adam Can the OpenShift CoCo offering with Azure CVM qualify to be mentioned here ?

Reply:
Yep, I've added it and also regarding AKS.
Can you run the AKS part with Amar?